AmplifyPlugins/Storage/AWSS3StoragePluginTests/Operation/AWSS3StorageUploadFileOperationTests2.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSS3StoragePlugin
+@testable import AWSPluginsCore
+import AWSS3
+
+final class AWSS3StorageUploadFileOperationTests2: AWSS3StorageOperationTestBase {
```

Comment:
I had to add an additional file because the linter was complaining that AWSS3StorageUploadFileOperationTests was too long.